### ISSUE TITLE
Getting 'unknown options: -noall_load' error

### ISSUE THREAD

**Issue by gcanessag** [optimizers/homebrew-cutest#65]
Trying to install pycutest through brew, pasting output:
```
% brew install cutest --without-single                                       
==> Downloading https://formulae.brew.sh/api/formula.jws.json
######################################################################################################################################################################################################################################################### 100.0%
==> Downloading https://formulae.brew.sh/api/cask.jws.json
######################################################################################################################################################################################################################################################### 100.0%
==> Fetching optimizers/cutest/cutest
==> Downloading https://github.com/ralna/CUTEst/archive/v2.0.2.tar.gz
Already downloaded: /Users/gcanessa/Library/Caches/Homebrew/downloads/19e24677b1620da0b5e5f79ce1bfc503b2eac3c4f76af8a0797076ef84b98116--CUTEst-2.0.2.tar.gz
==> Installing cutest from optimizers/cutest
==> Patching
==> ./install_cutest < cutest.input
==> gfortran -fPIC -shared -Wl,-all_load libcutest.a -Wl,-noall_load -o libcutest_double.dylib -Wl,-undefined -Wl,dynamic_lookup -headerpad_max_install_names
Last 15 lines from /Users/gcanessa/Library/Logs/Homebrew/cutest/02.gfortran:

gfortran
-fPIC
-shared
-Wl,-all_load
libcutest.a
-Wl,-noall_load
-o
libcutest_double.dylib
-Wl,-undefined
-Wl,dynamic_lookup
-headerpad_max_install_names

ld: unknown options: -noall_load 
collect2: error: ld returned 1 exit status

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/optimizers/homebrew-cutest/issues
```

**Comment by wangqiuoe** [comment]
Did you solve the problem?

**Comment by gcanessag** [comment]
Nope... I downloaded Docker and created a container. Did the calls and runs there and then exported the csv into the main drive. But to this day, no mac support unfortunately.

**Comment by jfowkes** [comment]
@dpo this is a bug with the homebrew-cutest installer on MacOS with gfortran-13:
```
==> Installing optimizers/cutest/cutest --without-single
==> Patching
==> ./install_cutest < cutest.input
==> gfortran -fPIC -shared -Wl,-all_load libcutest.a -Wl,-noall_load -o libcutes
Last 15 lines from /Users/runner/Library/Logs/Homebrew/cutest/02.gfortran:

gfortran
-fPIC
-shared
-Wl,-all_load
libcutest.a
-Wl,-noall_load
-o
libcutest_double.dylib
-Wl,-undefined
-Wl,dynamic_lookup
-headerpad_max_install_names

ld: unknown options: -noall_load 
collect2: error: ld returned 1 exit status
```

The `-noall_load` option is deprecated and should be removed as per:
https://stackoverflow.com/questions/32983398/what-is-noall-load-flag-replacement-on-mac-os-x

**Comment by jfowkes** [comment]
@amontoison can we get this fixed? `homebrew-cutest` is now broken due to this.

**Comment by amontoison** [comment]
You just need.to remove the flag `-noall_load`. It could be also easier to switch to the Meson build system.

**Comment by jfowkes** [comment]
@amontoison right but that needs to be done here in the homebrew recipe:
https://github.com/optimizers/homebrew-cutest/blob/master/cutest.rb#L70

**Comment by amontoison** [comment]
I opened #66.

**Comment by dpo** [comment]
Thank you for the report, and sorry for the delay. Fixed in #72.

**Comment by gcanessag** [comment]
It installed with no issues now, appreciate the help thanks so much!
